### DISCUSSION TITLE
Can we use Traefik plugins from the Traefik plugin marketplace (IP block list)?

### DISCUSSION THREAD

**Discussion by theliuk** [mia-platform/community#280] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v11

### Description

Hi,
Traefik offers a large number of plugins (middlewares) which are [documented here](https://doc.traefik.io/traefik/middlewares/overview/). Unfortunately, one official plugin that is missing ([because of reasons](https://github.com/traefik/traefik/pull/4454)) is the one which allows the user to define a block list of IPs. There's a plugin to define an allow list, but there isn't one to defined a block list.

However, I found out that there's a rich markeplace of community plugins for Traefik, [like this one](https://plugins.traefik.io/plugins/62947363ffc0cd18356a97d1/deny-ip-plugin).

The question is: can we use such plugins? I mean both technically and bureaucratically?

Thank you!

**Comment by davidebianchi** [comment, marked as answer]
Hi @theliuk! From the [Traefik pricing](https://traefik.io/pricing/) page, `Custom middlewares through public and private plugins` are available only with the enterprise plan. So, if you are using the open source one, I think you cannot use them. Otherwise, it should be possible.
